crates/bevy_pbr/src/material.rs

```diff
@@ -707,6 +708,15 @@ fn extract_mesh_materials<M: Material>(
 pub fn extract_entities_needs_specialization<M>(
     entities_needing_specialization: Extract<Res<EntitiesNeedingSpecialization<M>>>,
     mut entity_specialization_ticks: ResMut<EntitySpecializationTicks<M>>,
+    mut removed_view_visibility_components: Extract<RemovedComponents<ViewVisibility>>,
```

Comment:
I'd consider changing this to `Extract<RemovedComponents<MeshMaterial3d<M>>>` to prevent despawns of other materials / other entities affecting the performance of this method.

Reply:
Yeah, that's a good call.